src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -49,115 +52,57 @@ static char* xmlPayloadClean(const char*  payload, const char* payloadWord)
 
 /* ****************************************************************************
 *
-* postQueryContext -
+* queryForward - 
+*
+* An entity/attribute has been found on some context provider.
+* We need to forward the query request to the context provider, indicated in qcrsP->contextProvider
```

Comment:
Instead of using singular (e.g. "to the context provider"), it should be in plural ("to the context providers"). Not only this line, but all the preamble comment.